src/test/java/io/github/sskorol/converters/StringToIntegerConverter.java

```diff
@@ -0,0 +1,10 @@
+package io.github.sskorol.converters;
+
+import com.creditdatamw.zerocell.converter.Converter;
+
+public class StringToIntegerConverter implements Converter<Integer> {
```

Comment:
There's an existing [IntegerConverter](https://github.com/creditdatamw/zerocell/blob/master/core/src/main/java/com/creditdatamw/zerocell/converter/IntegerConverter.java#L6) in ZeroCell.